### ISSUE TITLE
Possibility to add imports "at the bottom of the imports"

### ISSUE THREAD

**Issue by buehler** [buehler/typescript-hero#97]
Related to #82 

When new imports are added at the top of the file, a new import should be at the bottom of the imports.

``` typescript
import ...
import ...
// << -- new import location

/* code */
```

**Comment by 19majkel94** [comment]
Must-have feature while using `reflect-metadata`.

Also it should divide external modules and internal app parts imports by empty line into two section and insert in correct section.
https://angular.io/styleguide#!#03-06

**Comment by jonscottclark** [comment]
I'd also like to see any automagically added imports added to the bottom of the list, but it should definitely be something configurable.

**Comment by buehler** [comment]
Will be resolved (I hope at least) with #102
